src/config/logic/RestrictionConfig.ts

```diff
@@ -0,0 +1,64 @@
+import {Restriction} from "../../datatypes/Restriction";
+import {ProjectElements, ProjectLinks, ProjectSettings, VocabularyElements} from "../Variables";
+import {parsePrefix} from "../../function/FunctionEditVars";
+import {addLink} from "../../function/FunctionCreateVars";
+import {getElemFromIRI, getNewLink} from "../../function/FunctionGetVars";
+import {LinkType} from "../Enum";
+import {Cardinality} from "../../datatypes/Cardinality";
+
+export var RestrictionConfig: {
+	[key: string]: (iri: string, restriction: Restriction) => string | void
+} = {};
+
+RestrictionConfig["http://www.w3.org/2002/07/owl#someValuesFrom"] = (iri, restriction) => {
+	return createConnection(iri, restriction, parsePrefix("owl", "allValuesFrom"));
+}
+
+RestrictionConfig["http://www.w3.org/2002/07/owl#allValuesFrom"] = (iri, restriction) => {
+	return createConnection(iri, restriction, parsePrefix("owl", "someValuesFrom"));
+}
+
+RestrictionConfig["http://www.w3.org/2002/07/owl#minQualifiedCardinality"] = (iri, restriction) => {
+	createCardinality(iri, restriction);
+}
+
+RestrictionConfig["http://www.w3.org/2002/07/owl#maxQualifiedCardinality"] = (iri, restriction) => {
+	createCardinality(iri, restriction);
+}
+
+function createConnection(iri: string, restriction: Restriction, pred: string) {
+	if (VocabularyElements[iri].restrictions.find(r => r.restriction === pred
```

Comment:
It may be better to use `lodash.find` for similar use cases: https://lodash.com/docs/4.17.15#find

The code would be something like:
```
const restrictions = VocabularyElements[iri].restrictions
if (find(restrictions, { restriction: pred, onProperty: restriction.onProperty, target: restriction.target })) { ... } 
```